app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  include Permissions::User
   # Include default devise modules. Others available are:
```

Comment:
If we are not using comments, I do not think we need them.

Reply:
Maybe there is a reason the comments were placed, the link to some reading resources sp I will leave them there for now

Reply:
I think they came auto-generated from the devise installer maybe - don't think they are doing too much harm there ...